pkg/sessions/redis/redis_store.go

```diff
@@ -148,7 +152,10 @@ func buildStandaloneClient(opts options.RedisStoreOptions) (Client, error) {
 			logger.Errorf("no certs appended, using system certs only")
 		}
 
-		opt.TLSConfig.RootCAs = rootCAs
+		/* #nosec */
+		opt.TLSConfig = &tls.Config{
```

Comment:
While they are probably mutually exclusive -- this setting will clobber the `tls.Config` you make above for `InsecureSkipVerify`.

Can we make sure to build a unified `tls.Config` that we attach to the redis opts?

Reply:
Already done when resolving @JoelSpeed's comment.